### DISCUSSION TITLE
NotExists Object Query

### DISCUSSION THREAD

**Discussion by adriancova** [dynamoose/dynamoose#1422] [Q&A] [answered]
Hi,

How to achieve an object query for condition not().exists() ? 

I've tried:

```
model.query({someField: {exists: false}}) // here it seems like the exists condition will be applied regardless of true/false/other value

model.query({someField: {notExists: false}}) // error: The type: notExists is invalid for the query operation
// same issue as before when trying with not_exists and not: 'exists'
```

at the moment I've solved this by using the .not().exists() functions but it'd be neat if I could wrap all the conditions in an object.

Thanks in advance for the help and extra thanks for the library, it has been really useful to me.

PS: I'm using 3.0.0-beta.1 version of dynamoose

**Comment by fishcharlie** [comment, marked as answer]
@adriancova This hasn't been implemented yet. Please feel free to create an issue on this repo for it. Or even better a pull request 😃
